.github/workflows/example_check_on_schedule.yml

```diff
@@ -24,7 +24,29 @@ jobs:
         all_loc=$( IFS=',' ; echo "${res[*]}" )
         echo "Found the examples: $all_loc"
         echo "matrix={\"directory\":$(echo "$all_loc")}" >> $GITHUB_OUTPUT
-
+  
+  prepare_cache_for_hf:
+    name: Prepare huggingface cache for example test
+    if: github.repository == 'hpcaitech/ColossalAI'
+    runs-on: [self-hosted, gpu]
+    container:
+      image: hpcaitech/pytorch-cuda:1.12.0-11.3.0
+      options: --rm
+    timeout-minutes: 5
+    defaults:
+      run:
+        shell: bash
+    concurrency:
+      group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-repare-cache
+      cancel-in-progress: true
+    steps:
+      - name: Prepare bert weights to cache_dir
+        run: |
+          pip install transformers
+          export model_name="bert-base-uncased"
+          export cache_dir="/home/lcsoftware/gitact/actions-runner/_work/_temp/_github_home/hf_cache"
```

Comment:
The path is different in container. See https://github.com/hpcaitech/ColossalAI/blob/4965c0dabd2df118a0ae33fd6f291fab617600a3/.github/workflows/build_on_pr.yml#L188